### PR TITLE
add kafka span decoder, tests, docs

### PR DESCRIPTION
**Pull request by jamescway**
- add back @fr0stbyte's test, make it test `KafkaProcessor`
- add a SpanDecoder, this is for use with zipkin-tracer gem

### CODE REVIEW

File: zipkin-receiver-kafka/src/test/scala/com/twitter/zipkin/receiver/kafka/KafkaProcessorSpec.scala [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+import com.twitter.zipkin.zookeeper.ZooKeeperClientFactory
+
+import java.util.Properties
+import com.twitter.app.{App, Flaggable}
+import com.twitter.zipkin.thriftscala
+
+@RunWith(classOf[JUnitRunner])
+class KafkaProcessorSpecSimple extends FunSuite with BeforeAndAfter {
+
+  val serializer = new BinaryThriftStructSerializer[thriftscala.Span] {
```
Comment (sprsquish) [inline zipkin-receiver-kafka/src/test/scala/com/twitter/zipkin/receiver/kafka/KafkaProcessorSpec.scala]: Use the one from SpanDecoder

File: zipkin-receiver-kafka/src/test/scala/com/twitter/zipkin/receiver/kafka/KafkaProcessorSpec.scala [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    def codec = thriftscala.Span
+  }
+
+  val topic = Map("integration-test-topic" -> 1)
+  val validSpan = Span(123, "boo", 456, None, List(new Annotation(1, "bah", None)), Nil)
+
+  case class TestDecoder extends Decoder[Option[List[ThriftSpan]]] {
+      val deserializer = serializer
```
Comment (sprsquish) [inline zipkin-receiver-kafka/src/test/scala/com/twitter/zipkin/receiver/kafka/KafkaProcessorSpec.scala]: Indent by two spaces, not four

**Comment by jamescway** [comment]
@sprsquish comments addressed

**Comment by sprsquish** [comment]
Thanks @jamescway. Pulled internally. Will show up here soon.

**Pull request closed without merging**